src/tracker/torrent.rs

```diff
@@ -93,7 +93,7 @@ impl Default for Entry {
 }
 
 #[derive(Debug)]
-pub struct Stats {
+pub struct Swarm {
```

Comment:
hi @da2ce7, just to confirm  you have used the term "Swarm" because in BitTorrent that means all the nodes that have the torrent.

https://en.wikipedia.org/wiki/Glossary_of_BitTorrent_terms#Swarm

Reply:
If this was a torrent client, I believe that should be something like "connections", as this is the index, the "swam" is all the peers that have this torrent. - So I believe it is the correct word.

Reply:
I just call it `Swam` instead of `SwamSize` or something, as we may add more info related to the Swam to this struct in the future.

Reply:
OK, I thought that was the reason. Maybe we can call it `SwamStats` now until we have more fields and then rename it. What do you think? I always try to reveal current intentions, not future behaviour, with names.